runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -346,6 +448,263 @@ else if (staminaTimer == null)
 			}
 		}
 
+		if (event.getVarbitId() == Varbits.ANTIFIRE && config.showAntiFire())
+		{
+			int antifireDuration = event.getValue() * 30;
+			Duration duration = Duration.of(antifireDuration, RSTimeUnit.GAME_TICKS);
+
+			if (antifireDuration == 0)
+			{
+				removeGameTimer(ANTIFIRE);
+				antifireTimer = null;
+			}
+			else if (antifireTimer == null)
+			{
+				antifireTimer = createGameTimer(ANTIFIRE, duration);
+			}
+			else
+			{
+				antifireTimer.updateDuration(duration);
+			}
+		}
+
+		if (event.getVarbitId() == Varbits.SUPER_ANTIFIRE && config.showAntiFire())
+		{
+			int superAntifireDuration = event.getValue() * 20;
+			Duration duration = Duration.of(superAntifireDuration, RSTimeUnit.GAME_TICKS);
+
+			if (superAntifireDuration == 0)
+			{
+				removeGameTimer(SUPERANTIFIRE);
+				superAntifireTimer = null;
+			}
+			else if (superAntifireTimer == null)
+			{
+				superAntifireTimer = createGameTimer(SUPERANTIFIRE, duration);
+			}
+			else
+			{
+				superAntifireTimer.updateDuration(duration);
+			}
+		}
+
+		if (event.getVarbitId() == Varbits.MAGIC_IMBUE && config.showMagicImbue())
+		{
+			int magicImbueDuration = event.getValue() * 10;
+			Duration duration = Duration.of(magicImbueDuration, RSTimeUnit.GAME_TICKS);
+
+			if (magicImbueDuration == 0)
+			{
+				removeGameTimer(MAGICIMBUE);
+				magicImbueTimer = null;
+			}
+			else if (magicImbueTimer == null)
+			{
+				magicImbueTimer = createGameTimer(MAGICIMBUE, duration);
+			}
+			else
+			{
+				magicImbueTimer.updateDuration(duration);
+			}
+		}
+
+		if (event.getVarbitId() == Varbits.DIVINE_SUPER_ATTACK && config.showDivine())
+		{
+			if (client.getVarbitValue(Varbits.DIVINE_SUPER_COMBAT) > 0)
```

Comment:
Rather than comparing to 0, this should be checking whether the value of scb is equal to super attack--if you pot a super attack after a scb, you'll have equal values for super strength, super defence, and scb, but a greater value for super attack as it will persist a little longer. This should be kept in mind for all potions which are combined in other potions.

Reply:
The changes in these varbit values don't occur at exactly the same time. As a result, checking whether the divine super attack varbit value equals the divine super combat varbit value within the block for the divine super attack varbit change event like you suggested fails.

Luckily, the varbits for the components of combination potions always seem to change before the varbits for the combination potions themselves (e.g. the divine super attack, strength, and defence varbits will change before the divine super combat potion varbit). Therefore, I was able to add equality checks to the combination potion varbit change event blocks instead.